---
name: Naveen
surname: Kannan
position: "Research Associate"
address: "Cleveland, Ohio | United States Permanent Resident"
www: naveenk2022.github.io/quarto_blog
phone: +1 973 519 5397
email: "naveen.kannan12@gmail.com"
github: naveenk2022
linkedin: naveen-kannan-177120118
date: "`r format(Sys.time(), '%B %Y')`"
aboutme: "Data Scientist and Bioinformatician with 2+ years of research experience applying advanced data analytics and machine learning to human genomic data. Experienced in automated deployment of scientific workflows with cutting edge machine learning, data analytics, and bioninformatics tools, with a strong passion for enabling easy access to Big Data tools and containers."
headcolor: 414141
output: 
  vitae::awesomecv
---
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
options(tinytex.verbose = TRUE)
library(vitae)
library(kableExtra)
library(here)
library(tibble)
library(readr)
library(glue)
source(file = here("resume/data_r","data_r.r"))
```

# Work Experience   

```{r Research, results = "asis"}
experience %>%
  detailed_entries(
    with = title,
    what = unit,
    why = detail,
    when = glue::glue("{startMonth} {startYear} - {endMonth} {endYear}",.na = ""),
    where = where, .protect = F
  )
```

# Skills 

```{r}
tribble(
   ~`Operating Systems`, 
   ~`Programming Languages`,
   ~`Software`,
   ~`Geospatial Analytics`,
   ~`Markup Languages`,
   ~`Version Control`,
   ~`Machine Learning`,
   ~`Bioinformatics Tools`,
  "Unix, Debian Linux, Fedora Red Hat Linux, Windows 11",
  "Python, R, Bash, SQL",
  "Hadoop/Spark, Hive, Jupyter, Docker, Singularity, SLURM, Ansible",
  "Geopandas, sf(R), QGIS, ArcGIS",
  "LaTeX, Markdown, RMarkdown",
  "Git, GitHub, gh client",
  "PySpark, Tensorflow, Pytorch, pandas, scikit-learn, numpy, scipy",
  "samtools, BCFtools, HTSlib, Tabix, Hail, Glow"
) %>%
  kable(booktabs = T, align = 'c', escape = F, format = "latex") %>%
  kable_styling(full_width = F, 
                font_size = 9, 
                latex_options = "hold_position", 
                position = "center") %>% 
  row_spec(0, bold = T, color = "#414141") %>% 
  row_spec(1, bold = F, color = "#7f7f7f") %>% 
  column_spec(column = 1:8, width = '1.75cm')
```


# Education

```{r}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where, ~ Detail,
  "MSc in Biomedical and Health Informatics", "JANUARY 2022 – AUGUST 2023", "Case Western Reserve University", "Cleveland, Ohio","GPA-3.9" ,
  "Bachelor of Medicine and Bachelor of Surgery", "SEPTEMBER 2016 - MARCH 2020", "Madras Medical College", "Chennai, India","GPA-4.0" 
) %>% 
  detailed_entries(Institution, Year, Degree, Where,Detail)
```

# Leadership

```{r Leadership, results = "asis"}

leadership %>%
  detailed_entries(
    with = title,
    what = unit,
    why = detail,
    when = glue::glue("{startMonth} {startYear} - {endMonth} {endYear}",.na = ""),
    where = where, .protect = F
  )

```

# Personal Projects

```{r PROJECTS}

projects %>%
  detailed_entries(
    with = area,
    what = accomplishment,
    why = detail,
    when = year,
    where = where, .protect = F
  )

```

```{r}
## For the following sections, note that:
# \bodyfont is a custom command that has been defined in the awesome-cv.cls file.
# \newcommand*{\bodyfont}{\sourcesanspro} 
# \bodyfontlight is similar
# If you want to download fonts and use them, you'd need to use fontspec
# For example, for the following if you used:
# \begingroup
# \fontspec [Path = fonts/] {Roboto-Regular}
# This would use the local Roboto font found in the fonts folder. The "/" IS NOT OPTIONAL.

```
# Publications

\begingroup
\bodyfontlight
\color{darktext}

- \small Ajayakumar, J., Curtis, A. J., Maisha, F. M., Bempah, S., Ali, A., \textbf{Kannan, N.}, Armstrong, G. & Morris Jr, J. G. (2024). Using spatial video and deep learning for automated mapping of ground-level context in relief camps. International Journal of Health Geographics, 23(1), 23.


\endgroup

\begingroup
\bodyfontlight
\color{darktext}

-   \small Ruksakulpiwat, S., Thongking, W., \textbf{Kannan, N.}, Wright, E., Niyomyart, A., Benjasirisan, C., ... & Still, C. H. (2024). Understanding the Relationship Between Comorbidities, Medication Nonadherence, Activities of Daily Living, and Heart Condition Status Among Older Adults in the United States: A Regression Analysis and Machine Learning Approach. Journal of Cardiovascular Nursing, 10-1097.


\endgroup

\begingroup
\bodyfontlight
\color{darktext}

- \small \textbf{Naveen Kannan}\textsuperscript{1}, Nicholas Wheeler\textsuperscript{1}, Genome Center for Alzheimer’s Disease, Li-San Wang\textsuperscript{2}, Yuk Yee Leung\textsuperscript{2}, William S. Bush\textsuperscript{1},\textsuperscript{1}Cleveland Institute for Computational Biology, Department for Population and Quantitative Health Sciences, Case Western Reserve University, Cleveland, Ohio 44106, USA;\textsuperscript{2} Department of Pathology and Laboratory Medicine, Penn Neurodegeneration Genomics Center, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, USA, Quality Control metrics of individual genetic variants in the Alzheimer’s Disease Sequencing Project are associated with FAVOR annotations.

\endgroup

\begingroup
\bodyfontlight
\color{darktext}

-   \small \textbf{N. Kannan}\textsuperscript{1}, N. Wheeler\textsuperscript{1}, Genome Center for Alzheimer's Disease, L-S. Wang\textsuperscript{2}, Y. Leung\textsuperscript{2}, W. S.
Bush\textsuperscript{1}; \textsuperscript{1}Cleveland Inst. for Computational Biology, Dept. for Population and Quantitative Hlth.Sci., Case Western Reserve Univ., Cleveland, OH, \textsuperscript{2}Dept. of Pathology and Lab. Med., Penn Neurodegeneration Genomics Ctr., Perelman Sch. of Med., Univ. of Pennsylvania, Philadelphia, PA., Annotation and scoring of the deleteriousness of individual genetic variants in the 4th release of the Alzheimer’s Disease Sequencing Project;(PB4451). Presented at the Annual Meeting of The American Society of Human Genetics, November 3, 2023, in Washington DC.

\endgroup

\begingroup
\bodyfontlight
\color{darktext}

-   \small Segamalai, D., Abdul Jameel, A. R., **Kannan, N**., Anbalagan, A., Duraisamy, B., Raju, P., & Devy Gounder, K. (2017). [Mediastinal pseudocyst: varied presentations and management---experience from a tertiary referral care centre in India.](https://downloads.hindawi.com/archive/2017/5247626.pdf) *HPB Surgery*, *2017*.

\endgroup

\begingroup
\bodyfontlight
\color{darktext}


-   \small Vellaisamy, R., **Kannan, N**., Anbalagan, A., Raju, P., Duraisamy, B., Murugesan, C. S., & Gounder, K. D. (2016). [Endoscopic access to hepatic duct through duodenum during follow up--after primary surgery for hepatolithiasis](https://hpbonline.org/article/S1365-182X(16)31428-9/fulltext). *HPB*, *18*, e530.

\endgroup

\begingroup
\bodyfontlight
\color{darktext}

-   \small Ramasamy, V., Vellaisamy, R., **Kannan, N**., & Gounder, K. D. (2016). [Refined technique of access loop in hepatobiliary surgery.](https://hpbonline.org/article/S1365-182X(16)31598-2/abstract) *HPB*, *18*, e593-e594.

\endgroup

\begingroup
\bodyfontlight
\color{darktext}

-   \small \textbf{Kannan, N.}, Vellaisamy, R., Govindarajan, M., & Gounder, K. D. (2016). [Pellagra following pancreaticoduodenectomy for malignant pancreatic carcinoid with pluripotent hormonal potential](https://www.hpbonline.org/article/S1365-182X(16)31013-9/fulltext). *HPB*, *18*, e381-e382.

\endgroup

